projects.js

```diff
@@ -0,0 +1,169 @@
+  });
+  // LOOPING TECHNOLOGIES
+  let technologies = '';
+  project.technologies.forEach((technology) => {
+    technologies += `<li>${technology}</li>`;
+  });
+  // WRITING HTML TEMPLATE
+  const code = `
+  <div class="projects container">
+  <div class="overflow">
+          <img class="project-img" src="${project.image}" alt="" />
+        </div>        
+        <div class="project-detail">
+          <h3 class="tertiary-heading">${project.name}</h3>
+          <ul class="project-details">
+            ${details}
+          </ul>
+          <p class="project-description">
+            ${project.description}
+          </p>
+          <ul class="project-lang">
+            ${technologies}
+          </ul>
+          <button class="btn" id="see-project-${index}">See project</button>
+        </div>
+        </div>
+  `;
+  template += code;
+});
+project.innerHTML = template;
+
+// POPUP window SECTION
+
+const popupContainer = document.createElement('div');
+popupContainer.classList.add('popup-container');
+
+function createPopup(project) {
+  let details = '';
+  project.details.forEach((detail) => {
+    details += `<li>${detail}</li>`;
+  });
+
+  let technologies = '';
+  project.technologies.forEach((technology) => {
+    technologies += `<li>${technology}</li>`;
+  });
+
+  const popupTemp = `
+    <div class="popup">
+      <div class="projects container popup-container">
+        <div class="project-title">
+          <h3 class="tertiary-heading">${project.name}</h3>
+          <button class="popup-close-btn">
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              fill="none"
+              viewBox="0 0 24 24"
+              stroke-width="1.5"
+              stroke="currentColor"
+              class="popup-close-icon"
+            >
+              <path
+                stroke-linecap="round"
+                stroke-linejoin="round"
+                d="M6 18L18 6M6 6l12 12"
+              ></path>
+            </svg>
+          </button>
+        </div>
+        <ul class="project-details">
+          ${details}
+        </ul>
+        <div class="overflow">
+          <img class="project-img popup-img" src="${project.image}" alt="" />
+        </div>
+        <div class="project-detail popup-details">
+          <p class="project-description popup-description">
+            ${project.description}
+          </p>
+          <div class="popup-technology">
+          <ul class="project-lang">
+            ${technologies}
+          </ul>
+          <div class="check-btn">
+            <button class="btn popup-btn">
+              <span>See live</span>
+              <a href="${project.liveLink}" target="_blank"><img class="popup-logo" src="images/see-live.svg" alt="" /></a>
+            </button>
+            <button class="btn popup-btn">
+              <span>See source</span>
+              <a href="${project.sourceLink}" target="_blank"><img class="popup-logo" src="images/source-logo.svg" alt=""></a>
```

Comment:
- [ ] It would be to make the button small a bit on mobile screens cause the button looks bigger than the ones in the Figma design so please make sure they are like the ones in the Figma design 
- Yours 
<img width="335" alt="Screenshot 2023-05-18 at 22 41 40" src="https://github.com/mahdinoori2000/portfolio/assets/61790587/3d17af61-2b71-4a48-a062-67be8a0ec5c3">

- Design 


<img width="318" alt="Screenshot 2023-05-18 at 22 43 33" src="https://github.com/mahdinoori2000/portfolio/assets/61790587/cbd32e0d-b4a4-44d5-a1e9-759360f246df">